noir_stdlib/src/hash/sha256.nr

```diff
@@ -169,99 +169,352 @@ unconstrained fn build_msg_block<let N: u32>(
     } else {
         BLOCK_SIZE
     };
-    for k in 0..block_input {
-        msg_block[k] = msg[msg_start + k];
+
+    // Figure out the number of items in the int array that we have to pack.
+    // e.g. if the input is [0,1,2,3, 4,5] then we need to pack it as 2 items: [0123, 4500]
```

Comment:
```suggestion
    // e.g. if the input is [0,1,2,3,4,5] then we need to pack it as 2 items: [0123, 4500]
```
nit, the space was bothering me lol

Reply:
Haha, ok, I wanted to make it easier to see what the grouping was.

Reply:
Ah I see what you were going for. I think it becomes clear in the separate to follow